Budgett/config.json

```diff
@@ -1,5 +1,9 @@
 {
     "prefix": "~",
     "generalChatID": "391063932535701505",
-    "testMode": "false"
+    "testMode": "false",
+    "wednesdayPath": "https://9gag.com/tag/it-is-wednesday-my-dudes",
+    "nGagPath": "https://9gag.com/gag/",
+    "defaultMeme": "https://www.youtube.com/watch?v=OzQ-KvxLVT0",
+    "STGID": "352255685183930369"
```

Comment:
Not a blocker but can we spell this out rather than using aberrations?

Reply:
Updated